### ISSUE TITLE
Cart Sort comment: melt inclusion specimen type

### ISSUE THREAD

**Issue by smrgeoinfo** [isamplesorg/metadata#15]
from '411411'
"not sure if melt inclusion is solid, liquid or gas, if liquid or gas, then container with fluid, otherwise piece of solid material"

**Comment by smrgeoinfo** [comment]
yes, really need better description of what the inclusion actually is.  In any case, I would think its safe to assume that the inclusion is a mineral grain (crystal) in a rock, and that the inclusion is a single object. This would lead to 'piece of solid material', problematic if the inclusion contains liquid or gas, but in that case it probably would be called a fluid inclusion.

**Comment by klehnert55** [comment]
Inclusions in the PetDB database are to the most part 'glass', so neither a mineral nor fluid nor gas.

**Comment by smrgeoinfo** [comment]
Fluid inclusions are an interesting problem.  Perhaps an edge case we can ignore (are they ever registered as identified specimens?).
analysis-- the inclusion is technically an object (under the definition of object as a 'thing that has a boundary the separates it from other things in the universe'), but it is not solid, so not piece of solid material. I suppose one could imagine a specimen of a particular pore space in a rock or bone as an analogous thing.  For now I'd suggest if we do have 'fluid inclusion' type specimens, they could be categorized as container with fluid, where the container is a natural object (e.g. a mineral grain in a rock), not a human-made object (e.g. a bottle).
